doc/themeCustomization.md

```diff
@@ -2,76 +2,93 @@
 
 ## Create a new color theme
 
-- First, duplicate directory [/src/theme/custom/default](../src/theme/custom/default) and rename it as your wish,
-  exemple('fooDark', 'fooLight', etc.).
+- First, duplicate the directories [/src/theme/custom/cosmoDark](../src/theme/custom/cosmoDark)
+ (resp. [/src/theme/custom/cosmoLight](../src/theme/custom/cosmoLight)) and rename it fooDark (resp. fooLight) for instance.
 
-- Then, in the newly created dircetory open the matching file [/src/theme/custom/fooDark/palette.js](../src/theme/custom/fooDark/palette.js) and
-  set the colors dictionnary:
-
-```javascript
-const themeColors = {
-  primary: '#FFAD38',
-  primaryVariant: '#A16612',
-  secondary: '#466180',
-  background: '#191919',
-  backgroundVariant: '#1F1F1F',
-  surface: '#2E2E2E',
-  error: '#FF667F',
-  warning: '#FB8C00',
-  success: '#19E152',
-  info: '#1E88E5',
-  white: '#FFFFFF',
-  black: '#000000',
-  grey: '#999A9D',
-  backgroundSignInPage: '#2f363b',
-  backgroundSignInButton: '#2F2F2F',
-};
-```
-
-- Then you can recompose the correspondance between color and display elements:
+- Then, in the newly created directory, open [/src/theme/custom/fooDark/palette.js](../src/theme/custom/fooDark/palette.js)
```

Comment:
I'm not sure it's a good idea to provide a link to non-existing page, maybe we can just give the path?

Reply:
I kept the existing links, I can remove them, I didn't test the behavior regarding those links.

Reply:
+1 for showing only the path, instead of broken links :slightly_smiling_face: